kaguna/dimba-api
# #160742080 Authenticate and Authorize users operations

#### What does this PR do?
- Authenticates and Authorizes users operations in the controllers.

#### Description of Task to be completed?
- Create roles table
- Install [pundit](https://github.com/varvet/pundit) to control user operations.
- Authorize all the controllers.

#### How should this be manually tested?
- Login using user credentials and try perform actions for the admin e.g. creating a new league.
- `Unauthorized` error should pop up. 

#### Pt story?
- [#160742080](https://www.pivotaltracker.com/story/show/160742080)
